app/assets/javascripts/govuk_publishing_components/components/accordion.js

```diff
@@ -55,6 +56,16 @@ window.GOVUK.Modules.GovukAccordion = window.GOVUKFrontend.Accordion;
         console.error('Could not read accordion data attributes error: ' + e.message, window.location)
```

Comment:
This code isn't part of the PR but I just noticed it and wanted to check - is this meant to be here?

Reply:
Yes, it is. In the event that incorrect data is passed to the component we want an error to happen in the console.

Ideally it would be noticed by the developer implementing it, but we also have stuff to monitor this that should help us identify configuration problems like this.